package/src/views/SkiaView.web.tsx

```diff
@@ -82,13 +81,13 @@ export class SkiaView extends React.Component<
    */
   private redraw() {
     if (this._mode === "continuous" || this._redrawRequests > 0) {
-      this._redrawRequests = 0;
```

Comment:
This is the main issue I was hitting: the redrawRequest was made by onLayout but at this point the size still shows -1. Only in the frame after it was available. So I thought we would cancel redraw requests only if we draw.

The issue is only reproducible in Remotion, not sure why.